File: test/clients/echo.py
import json
from pathlib import Path
from anvil.server import call, connect
from tools import rpc


UTF_8 = "utf-8"

connect(
    (json.loads((Path.cwd() / "secrets.json").read_text(encoding=UTF_8)))[
        "development"
    ]["client"]
)



response = rpc.echo(42)

print('response:', response)

File: test/client_code/tests/client/rpc/echo.js
/*
client/rpc/echo.js
*/

export default async () => {
  await (async () => {
    const echo = await use("rpc/echo");
    const response = await echo({ foo: 420 }, 10, 20, 30);
    console.log("response:", response);
  })();

  await (async () => {
    
  })();

  

};
